### PR TITLE
Deprecate SRID value `variable`

### LINKED ISSUE #1935: CSDL: deprecate SRID value "variable"
With #456 we removed mention of the obsolete [GeoJSON-2008] document and the optional [CRS object](http://geojson.org/geojson-spec.html#named-crs).

This makes it non-obvious/impossible to serialize Geo values of properties with `SRID="variable"`.

### Proposal

CSDL
- Deprecate (or remove?) [SRID](https://docs.oasis-open.org/odata/odata-csdl-xml/v4.02/csd01/odata-csdl-xml-v4.02-csd01.html#SRID) value `variable`
- Mention in "Changes" section

JSON Format
- Mention #456 in "Changes" section
- Refer to https://datatracker.ietf.org/doc/html/rfc7946#section-4 in paragraph about Geo serialization

### PR DESCRIPTION
**Pull request by ralfhandl**
Fixes #1935

### CODE REVIEW

**Review by sthagen — APPROVED** [review]
LGTM, thanks a lot. Much appreciated.

**Review by HeikoTheissen — CHANGES_REQUESTED** [review]

File: odata-json-format/1 Introduction.md [hunk 1]
```diff
@@ -24,6 +24,9 @@ Section | Feature / Change | Issue
 [Section ##ControlInformationmediaodatamedia]| 
  `mediaContentType` can be `null`| 
 [536](https://github.com/oasis-tcs/odata-specs/issues/536)
+[Section ##StructuralProperty], [Section ##InformativeReferences]|
```
[the remark this answers is not in the document]
Comment (ralfhandl) [reply odata-json-format/1 Introduction.md:27]: 4.01 referenced GeoJSON-2008, which we removed: https://docs.oasis-open.org/odata/odata-json-format/v4.01/odata-json-format-v4.01.html#sec_NonNormativeReferences

File: odata-csdl/1 Introduction.md [hunk 2]
```diff
@@ -32,6 +32,9 @@ Section | Feature / Change | Issue
 [Section ##PrimitiveTypes]| 
 Allow stream-valued non-binding parameters| 
 [525](https://github.com/oasis-tcs/odata-specs/issues/525)
+[Section ##SRID]| 
+SRID value `variable` is deprecated| 
+[ 1935](https://github.com/oasis-tcs/odata-specs/issues/1935)
```
[the remark this answers is not in the document]
Comment (HeikoTheissen) [reply odata-csdl/1 Introduction.md:37]: This is not resolved.

**Review by HeikoTheissen — APPROVED** [review]

**Comment by ralfhandl** [comment]
Rebuilt after merging `main`

**Review by sthagen — APPROVED** [review]
Ship it!

**Review by sthagen — APPROVED** [review]
Getting better all the time. Thanks.

File: odata-csdl/1 Introduction.md [hunk 3]
```diff
@@ -525,10 +528,11 @@ underlying type is `Edm.Stream`, cannot be used in collections.
...
 
-See rule `primitiveLiteral` in [OData-ABNF](#ODataABNF) for the
-representation of primitive type values in URLs and
-[OData-JSON](#ODataJSON) for the representation in requests and
-responses.
+Representation of primitive type values is usage specific, see for example
+- rule `primitiveLiteral` in [OData-ABNF](#ODataABNF) for the
+representation in URLs, and
+- [OData-JSON](#ODataJSON) for the representation in JSON request and
+response bodies.
```
Comment (ralfhandl) [inline odata-csdl/1 Introduction.md:535]:
```suggestion
Representation of primitive type values within a URL is defined by the rule `primitiveLiteral` in [OData-ABNF](#ODataABNF).
Representation within request and response bodies is format specific.
```

File: odata-csdl/1 Introduction.md [hunk 4]
```diff
@@ -812,6 +813,8 @@ spatial reference system is applied to its values.
 The value of the `SRID` facet MUST be a non-negative integer or the
 special value `variable`. If no value is specified, the facet defaults
 to `0` for `Geometry` types or `4326` for `Geography` types.
+Services SHOULD NOT use the special value `variable` as, for example, the [OData-JSON](#ODataJSON)
+format does not define a representation for instance-specific spatial reference systems.
```
Comment (ralfhandl) [inline odata-csdl/1 Introduction.md:817]:
```suggestion
Services SHOULD NOT use the special value `variable` as some formats, for example [OData-JSON](#ODataJSON), do not define a representation for instance-specific spatial reference systems.
```

**Review by HeikoTheissen — APPROVED** [review]

**Review by mikepizzo — COMMENTED** [review]
I don't know that the reference to ODATA-JSON is required, but okay if others feel strongly.

**Pull request merged**